packages/catalog-readmodel-writer/src/model/converterV2.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  AgreementApprovalPolicy,
+  AgreementApprovalPolicyV2,
+  EServiceAttribute,
+  Descriptor,
+  DescriptorState,
+  Document,
+  EService,
+  EServiceAttributeV2,
+  EServiceDescriptorStateV2,
+  EServiceDescriptorV2,
+  EServiceDocumentV2,
+  EServiceTechnologyV2,
+  EServiceV2,
+  Technology,
+  agreementApprovalPolicy,
+  descriptorState,
+  technology,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { P, match } from "ts-pattern";
+
+export const fromAgreementApprovalPolicyV2 = (
+  input: AgreementApprovalPolicyV2 | undefined
+): AgreementApprovalPolicy | undefined => {
+  if (input == null) {
+    return undefined;
+  }
+
```

Comment:
being mandatory, can this be changed to
```suggestion
  input: AgreementApprovalPolicyV2
): AgreementApprovalPolicy => {
```
?